### PR TITLE
List against

### PR DESCRIPTION
**Pull request by asmeurer**
Fix sympy-bot list to show the branch the PR is against.

Also fix `sympy-bot review all` and `sympy-bot review mergeable` (and fix the spelling of "mergeable"!).

### CODE REVIEW

**Comment by asmeurer** [comment]
Ah, I thought I had the latest master when I did this, but I guess the pull didn't work.  So the mergeable fix from Julien is used instead (though it was probably too kind to still allow "mergable").

File: utils/github.py [hunk 1]
```diff
@@ -114,36 +114,46 @@ def github_list_pull_requests(urls, numbers_only=False):
...
+        formatted_pulls.append({
+            'created_at': created_at,
+            'n': n,
+            'repo': repo,
+            'branch': branch,
+            'author': author,
+            'mergeable': mergeable,
+            'branch_against': branch_against,
+        })
+    formatted_pulls.sort(key=lambda x: x['created_at'])
```
Comment (jrioux) [inline utils/github.py:127]: I would sort by `int(x['n'])` instead of by date.

**Comment by jrioux** [comment]
Looks fine.

File: utils/github.py (hunk 1 above)
Comment (asmeurer) [inline utils/github.py:127]: That's just how it was.

**Comment by jrioux** [comment]
It's in.

**Pull request merged**